src/main/java/com/google/devtools/build/lib/analysis/BuildView.java

```diff
@@ -430,6 +431,24 @@ public AnalysisResult update(
       skyframeBuildView.clearInvalidatedConfiguredTargets();
     }
 
+    TopLevelConstraintSemantics topLevelConstraintSemantics =
+        new TopLevelConstraintSemantics(
+            skyframeExecutor.getPackageManager(),
+            input -> skyframeExecutor.getConfiguration(eventHandler, input),
+            eventHandler);
+
+    TopLevelConstraintSemantics.PlatformRestrictionsResult platformRestrictions =
+        topLevelConstraintSemantics.checkPlatformRestrictions(
+            skyframeAnalysisResult.getConfiguredTargets(), explicitTargetPatterns, keepGoing);
+
+    if (platformRestrictions.targetsWithErrors.size() > 0) {
```

Comment:
Nit: Replace with the simpler "platformRestrictions.targetsWithErrors.isEmpty()".

Reply:
Done